core-java-modules/core-java-dates-operations/src/main/java/com/baeldung/date/comparison/DateComparisonUtils.java

```diff
@@ -0,0 +1,38 @@
+package com.baeldung.date.comparison;
+
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.Calendar;
+import java.util.Date;
+
+import org.apache.commons.lang3.time.DateUtils;
+

```

Comment:
Formatting is out. Please use for example provided formatter:
- https://github.com/eugenp/tutorials/blob/master/eclipse/formatter.xml